inventories.go

```diff
@@ -52,3 +55,57 @@ func (i *InventoriesService) CreateInventory(data map[string]interface{}, params
 
 	return result, nil
 }
+
+// UpdateInventory update an awx inventory
+func (i *InventoriesService) UpdateInventory(id int, data map[string]interface{}, params map[string]string) (*Inventory, error) {
+	result := new(Inventory)
+	endpoint := fmt.Sprintf("/api/v2/inventories/%d", id)
+	payload, err := json.Marshal(data)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := i.client.Requester.PatchJSON(endpoint, bytes.NewReader(payload), result, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := CheckResponse(resp); err != nil {
+		return nil, err
+	}
+
+	return result, nil
+}
+
+// GetInventory retrives the inventory information from its ID or Name
+func (i *InventoriesService) GetInventory(id int, params map[string]string) (*Inventory, error) {
+	endpoint := fmt.Sprintf("/api/v2/inventories/%d", id)
+	result := new(Inventory)
+	resp, err := i.client.Requester.GetJSON(endpoint, result, map[string]string{})
+	if err != nil {
+		return nil, err
+	}
+
+	if err := CheckResponse(resp); err != nil {
+		return nil, err
+	}
+
+	return result, nil
+
+}
+
+// Delete an inventory from AWX
+func (i *InventoriesService) Delete(id int) (*Inventory, error) {
```

Comment:
I suggest change the func name to `DeleteInventory`, as we have lots of test cases, and the same fun name could broken the whole test.